Makefile

```diff
@@ -77,7 +77,7 @@ endif
 BASEIMAGE?=gcr.io/google-containers/debian-base-$(ARCH):0.3
 
 GO_BUILD       = env CGO_ENABLED=0 GOOS=$(PLATFORM) GOARCH=$(ARCH) \
-                  go build -i $(GOFLAGS) -a -tags netgo -installsuffix netgo \
+                  go build $(GOFLAGS) -a -tags netgo -installsuffix netgo \
```

Comment:
Why did you remove -i, was that part of your testing out go 1.10 builds? Or something else?

Reply:
oops, accidentally added this to an existing PR rather than a nw one.

Reply:
from https://github.com/kubernetes-incubator/service-catalog/pull/2180